Gruntfile.js

```diff
@@ -127,5 +127,5 @@ module.exports = function(grunt) {
 
   grunt.registerTask('build', ['jshint', 'clean', 'concat', 'string-replace', 'uglify']);
 
-  grunt.registerTask('default', ['compile']);
+  grunt.registerTask('default', ['watch', 'compile']);
```

Comment:
What does adding the watch task alongside compile do here?

Reply:
Was used during testing to speed up development, have removed this 👍